README.md

```diff
@@ -6,63 +6,63 @@
 
 
 
-## What this is
+## Universal Idea
 
-Mandelbulber generates 3-dimensional fractals.
+In summary, Mandelbulber generates 3-dimensional fractals.
 
-Explore trigonometric, hypercomplex, mandelbox, IFS and many other 3D fractals.
+Explore trigonometric, hyper-complex, Mandelbox, IFS, and many other 3D fractals.
 
-Render with a great palette of customized shaders and create stunning images and videos.
```

Comment:
My two cents. Drop the shade/shaders.  
Render with a great palette of customizable materials to  create stunning images and videos.